bindings_test.go

```diff
@@ -56,72 +53,37 @@ func TestFanout(t *testing.T) {
 }
 
 func TestTopic(t *testing.T) {
-	wasCalled := map[string]chan string{
-		"foo.#": make(chan string),
-		"foo.*": make(chan string),
-		"baz.*": make(chan string),
-	}
-
-	s := NewServer(bindingsTestURL)
-	c := NewClient(bindingsTestURL)
+	s := NewServer(serverTestURL)
+	c := NewClient(serverTestURL)
 	defer c.Stop()
 
+	wasCalled := make(chan struct{})
+
 	s.Bind(TopicBinding("", "foo.#", func(ctx context.Context, rw *ResponseWriter, d amqp.Delivery) {
-		wasCalled["foo.#"] <- string(d.Body)
-	}))
-	s.Bind(TopicBinding("", "foo.*", func(ctx context.Context, rw *ResponseWriter, d amqp.Delivery) {
-		wasCalled["foo.*"] <- string(d.Body)
-	}))
-	s.Bind(TopicBinding("", "baz.*", func(ctx context.Context, rw *ResponseWriter, d amqp.Delivery) {
-		wasCalled["baz.*"] <- string(d.Body)
+		wasCalled <- struct{}{}
 	}))
 
 	stop := startAndWait(s)
 	defer stop()
 
-	cases := []struct {
```

Comment:
Are all these cases still covered? Shouldn't this be a separate commit for something else (other than ack)?

Reply:
Yes! There where some problems with the way this test was implemented and it was causing slowdowns. We don't really need to test AMQP, we just need to make sure we did the binding correctly. This proves that because only a topic exchange behaves this way.

But I could put it in another PR if you want.

Reply:
Ok, cool! Nah keep it in this PR! Was just the commit (message) I was nit-picking about, but no biggie. Maybe this resolves #66? Would be great!

Reply:
It should, but let's see.